sources/targets/Stride.Core.props

```diff
@@ -223,7 +223,7 @@
 
   <!-- SourceLink -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Condition="'$(StrideProjectType)' == 'CSharp'" Version="1.0.0" PrivateAssets="All"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Condition="'$(StrideProjectType)' == 'CSharp'" VersionOverride="1.0.0" PrivateAssets="All"/>
```

Comment:
Note: this is a bit hackish but necessary because the `Directory.Packages.props` gets imported later, and the build complains if we try to use `Version` nonetheless.